ONSdigital/census-rm-ops
# update development config to use the correct queue for the sample load

# Motivation and Context
The queue used to ingest samples has changed to "case.sample.inbound"

# What has changed
The default queue name used in the development config has been updated to "case.sample.inbound"

# How to test?
setup RM locally via census-rm-docker-dev, load a sample via the ops tool and cases should get created
